README.md

```diff
@@ -4,10 +4,10 @@
  
 # pretty-print - pp
 
-Adds a `pp(Any?)` and `T.pp()` method to pretty print any Java or Kotlin object.
+Adds a `pp(Any?)` and `T.pp()` method to pretty print Java or Kotlin object.
```

Comment:
I would rather have it say `any` here. Without the `any` it would also need to be plural, which would indicate that the method could print multiple objects at a time, which it does not do.